pypika/terms.py

```diff
@@ -340,19 +345,23 @@ def __init__(self, term, container, alias=None):
             The term to assert membership for within the container.
         :param container:
             A list or subquery.
+        :param isin:
+            A boolean to alter the behaviour to not in if false.
         """
         super(ContainsCriterion, self).__init__(alias)
         self.term = term
         self.container = container
+        self.isin = isin
```

Comment:
It would probably make sense to pull this up to the base class Criterion since any sort of criterion could be negated